app/script/view_model/ParticipantsViewModel.js

```diff
@@ -189,6 +192,11 @@ z.ViewModel.ParticipantsViewModel = class ParticipantsViewModel {
       return this.stateParticipants() && this.selectedUser() && !this.selectedService();
     });
 
+    this.selectedIsInConversation = ko.observable(false);
```

Comment:
This should be a computed. Otherwise the UI will not change when the group participants change.